### ISSUE TITLE
Allow polling parameter to be set during agent setup

### ISSUE THREAD

**Issue by AnneThKu** [org-arl/fjage-sentuator#1]
The config parameter `poll` was set in the class as given below.
```
import org.arl.fjage.sentuator.*

class PrSentuator extends Sentuator {

  // define your measurements
  static class MyMeasurement extends Measurement {
    float pressure
  }

  void setup() {
    sentuatorName = 'Pressure Sentuator'    // name your sentuator
    config.ofs = 100  
    config.poll = 3000                  // setup your configuration parameters
  }

  Measurement measure() {
    // generate a random measurement with the configured offset
    return new MyMeasurement(pressure: Math.random() + config.ofs)
  }

}
```
But after loading the agent (named as `prsen`), the `poll` parameter value is shown as 0.
![image](https://user-images.githubusercontent.com/42397354/66101016-daa19100-e5df-11e9-9b1f-2055fd9e9c2a.png)

Polling works when the parameter value is set from the shell.
![image](https://user-images.githubusercontent.com/42397354/66101142-4a178080-e5e0-11e9-9e4b-25e3b0ecf9f2.png)

**Comment by mchitre** [comment]
`poll` is a special configuration entry that is only available after initialization, and available via the shell. Will think through how to make it available programmatically in the agent, even before startup.

**Comment by notthetup** [comment]
```
  void setup(){
    sentuatorName = 'Depth Sensor Sentuator'
    setPollingInterval(5000)
  }
```

This seems to work for now. :)

**Comment by mchitre** [comment]
Should be fixed in 1.1.0
